proposals/refactor.md

```diff
@@ -0,0 +1,151 @@
+# Separate Components from Containers.
+
+## Problem
+
+We are thinking about moving from Redux. The first step will be refactoring our components and separate presentation from data management.
+This will result in components and containers, containers manage data from the store and dispatch it to components, that only deal with the presentation.
+This will allow us the proceded with the second step: refactor containers so we can remove the connection with redux and use the standard React state instead.
+
+## Solution
+
+### Components with few properties from store
+
+For all those components which take only few properties from the store, we can pass it as a props from component above.
+Component will be connect only with translations and will have all CSS styles there too.
+
+```js
+// components/MyComponent/index.jsx
+
+import React from 'react';
+import { translate } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+ MyComponent.propTypes = {
+  ...<YOUR PROPS GO HERE>
+};
+
+MyComponent.defaultProps = {
+  ...<YOUR DEFAULT PROPS GO HERE>
+};
+
+ export function MyComponent({PROPS}) {
+  return (
+      ....<YOUR CODE GO HERE/>
+  );
+}
+ export default translate(['translation_json'])(MyComponent);
```

Comment:
translation should be done on `scene` level

Reply:
@cutandpastey what do you think?

Reply:
I think @emuraton is correct.